packages/editor-ui/src/components/Node.vue

```diff
@@ -16,6 +16,11 @@
 				</n8n-tooltip>
 			</div>
 
+			<n8n-tooltip placement="top" :manual="true" :value="showWebhookNodeTooltip">
```

Comment:
<img width="441" alt="Screen Shot 2021-11-15 at 10 37 25 AM" src="https://user-images.githubusercontent.com/4711238/141758109-1080cb91-162c-4226-a136-b19cd5799c13.png">
based on designs, might need to set a max width on the tooltip

Reply:
This is also fixed. I created one class node-tooltip. Wanted to put inside the main class .node-wrapper, but that way I could override the default max-width only with !important which it is not good solution. Therefore I put the class in scss outside the main .node-wrapper class.